common-ui/common-ui-devstart.rb

```diff
@@ -41,6 +41,8 @@ def build(cmd_name, ui_name, args)
   options = BuildOptions.new.parse(cmd_name, args)
 
   common = Common.new
+
+  common.run_inline %W{gsutil cp gs://all-of-us-workbench-test-credentials/.npmrc .}
```

Comment:
Should this copy to `..` instead? I thought the other commands placed this at the git repo root

Reply:
ope, yes it should, I was imagining this being run at the repo root but I guess all the other yarn commands run from `/api`.